go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/ONSdigital/dp-publishing-dataset-controller
 go 1.13
```

Comment:
Sorry just noticed this, it probably should be bumped to 1.15

And the `ci/build.yml` and `ci/unit.yml` will need to be updated to go version 1.15.7

Reply:
👍